File: output/qwen_2.5_72b_instruct/avatar/direct_translation/java/python/successful_translation/atcoder_ABC042_A.py

import sys

input = sys.stdin.read().split()
x = int(input[0])
a = int(input[1])
y = int(input[2])
h = "NO"
if x == 5 or x == 7 or a == 5 or a == 7 or y == 5 or y == 7:
    if x + y + a == 17:
        h = "YES"
print(h)

File: output/gpt_4o/avatar/algo_based_translation/python/java/atcoder_ABC042_A.java

import java.util.Scanner;

public class atcoder_ABC042_A {
    public static void main(String[] args) {
        processInput();
    }

    public static void processInput() {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        scanner.close();

        int[] numbers = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            numbers[i] = Integer.parseInt(input[i]);
        }

        int countOfFives = 0;
        int sum = 0;
        for (int number : numbers) {
            if (number == 5) {
                countOfFives++;
            }
            sum += number;
        }

        if (countOfFives == 2) {
            if (sum == 17) {
                System.out.println("YES");
            } else {
                System.out.println("NO");
            }
        } else {
            System.out.println("NO");
        }
    }
}

